util/types/time_test.go

```diff
@@ -48,6 +48,8 @@ func (s *testTimeSuite) TestDateTime(c *C) {
 		{"20121231113045", "2012-12-31 11:30:45"},
 		{"121231113045", "2012-12-31 11:30:45"},
 		{"2012-02-29", "2012-02-29 00:00:00"},
+		{"00-00-00", "0000-00-00 00:00:00"},
+		{"00-00-00 00:00:00.123", "2000-00-00 00:00:00"},
```

Comment:
Because `ParseDataTime` use DefaultFsp which is 0, fsp part is ignored. @XuHuaiyu